core/sql/ustat/hs_cli.cpp

```diff
@@ -5442,12 +5427,17 @@ NAString HSSample::getTempTablePartitionInfo(NABoolean unpartitionedSample,
 //
 void HSSample::addTruncatedSelectList(NAString & qry)
   {
+    bool first = true;
     for (Lng32 i = 0; i < objDef->getNumCols(); i++)
       {
-        if (i)
-          qry += ", ";
+        if (!ComTrafReservedColName(*objDef->getColInfo(i).colname))
```

Comment:
I have written https://issues.apache.org/jira/browse/TRAFODION-2333 to cover this follow-on work. Thanks for the idea.